src/main/java/page_objects/LoginPage.java

```diff
@@ -25,6 +25,15 @@ public void login(Account account) {
         this.btnLogin.click();
     }
 
+    public void loginNTimes(Account account, int n) {
+        for(int i=0;i<n;i++){
```

Comment:
spacing

Reply:
Thank you for review, i Iwill format the code now